torchvision/csrc/io/image/image.cpp

```diff
@@ -27,7 +27,10 @@ static auto registry = torch::RegisterOperators()
                            .op("image::read_file", &read_file)
                            .op("image::write_file", &write_file)
                            .op("image::decode_image", &decode_image)
-                           .op("image::decode_jpeg_cuda", &decode_jpeg_cuda);
+#ifdef WITH_NVJPEG
+                           .op("image::decode_jpeg_cuda", &decode_jpeg_cuda)
+#endif
```

Comment:
This is rather an uncommon idiom for the code-base. If you see the rest of the IO methods (like _jpeg/_png) they are always defined but throw a runtime error if used:
https://github.com/pytorch/vision/blob/da7680f027762577d84577b59f41221cff07d341/torchvision/csrc/io/image/cpu/decode_jpeg.cpp#L7-L14